Blueprints/BlueprintDefinitions/vs2022/SimpleDynamoDBFunction/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs

```diff
@@ -1,11 +1,9 @@
 using Xunit;
```

Comment:
This file has significant changes to deal with change to move to new major version of Amazon.Lambda.DynamoDBEvents which removed the dependency on the SDK.